homeassistant/components/input_datetime/services.yaml

```diff
@@ -4,11 +4,11 @@ set_datetime:
     entity_id: {description: Entity id of the input datetime to set the new value.,
       example: input_datetime.test_date_time}
     date: {description: The target date the entity should be set to. Do not use with datetime.,
-      example: '"date": "2019-04-22"'}
+      example: '"2019-04-20"'}
```

Comment:
So the issue with this file is that someone has copy pasted JSON into the YAML. It's valid yaml but also ugly to read. 

Could you convert all the JSON to YAML syntax?

Reply:
Sure, don't know how that got past me. I removed everything locally, but removing the extra quotes from date and datetime does some weird things, I don't know why but this is how it looks if i fill in the sample data espaced only with double quotes (like this`example: "2019-04-20"`)

![image](https://user-images.githubusercontent.com/12055376/73128326-d0136900-3fcd-11ea-80cb-4b65f9395905.png) 

Yaml seems to convert the date that, which doesn't work in this case (fails to parse the date because it doesn't expect a time component)

Reply:
Oh, in that case just remove the extra pair of quotes. Right now it has 2 quotes around it.